daemon/daemon_linux_test.go

```diff
@@ -343,3 +348,66 @@ func TestRootMountCleanup(t *testing.T) {
 		assert.Assert(t, d.cleanupMounts())
 	})
 }
+
+func TestIfaceAddrs(t *testing.T) {
+	CIDR := func(cidr string) *net.IPNet {
+		t.Helper()
+		nw, err := types.ParseCIDR(cidr)
+		assert.NilError(t, err)
+		return nw
+	}
+
+	for _, tt := range []struct {
+		name string
+		nws  []*net.IPNet
+	}{
+		{
+			name: "Single",
+			nws:  []*net.IPNet{CIDR("172.101.202.254/16")},
+		},
+		{
+			name: "Multiple",
+			nws: []*net.IPNet{
+				CIDR("172.101.202.254/16"),
+				CIDR("172.102.202.254/16"),
+			},
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Do we need to capture `tt` here in this loop? (`tt := tt`)

Reply:
No, because `t.Parallel()` is not called. [`go vet` now knows about `t.Parallel()`](https://github.com/golang/tools/commit/d5e9e3592c8a808278ce1178637742b6ea320708) so we no longer have to preemptively do so "just in case" the `t.Parallel()` is added in later.